src/react-demo/Icons.es6.js

```diff
@@ -20,7 +20,7 @@ import {
   Website, Chat, RotateForward, Globe, At, Target, IconRow, IconColumn, IconSection, FolderFilled, BlogPost, MenuItems, Popup, Bold, Italic, Underline, UnorderedList, OrderedList,
   Flag, Testimonials, Chart, Play, Pause, AlignLeftV2, AlignCenterV2, AlignRightV2, AlignJustifyV2, Sidebar, ArrowCircle, Palette, Lightbox, Filter, Slider,
```

Comment:
You will need to update the version in package.json file. Murat missed it in Zap icon PR and we had to bump the version separately. But here is PR which bumps the version for reference -- https://github.com/b12io/metronome/pull/13/files